lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/view/clazzassignment/detailoverview/ClazzAssignmentDetailOverviewScreen.kt

```diff
@@ -185,39 +187,32 @@ fun ClazzAssignmentDetailOverviewScreen(
         }
 
         item(key = "deadline") {
-            if (uiState.cbDeadlineDateVisible){
-                ListItem(
-                    leadingContent = {
-                        Icon(
-                            Icons.Filled.EventAvailable,
-                            contentDescription = null
-                        )
-                    },
-                    headlineContent = { Text(formattedDateTime)},
-                    supportingContent = { Text(stringResource(MR.strings.deadline)) }
-                )
+            if (uiState.cbDeadlineDateVisible) {
```

Comment:
Revert what was not changed: This code section should not have been changed and now the indentation is wrong.